client/src/main/java/com/kritsit/casetracker/client/domain/ui/controller/ChangePasswordController.java

```diff
@@ -0,0 +1,73 @@
+package com.kritsit.casetracker.client.domain.ui.controller;
+
+import com.kritsit.casetracker.client.domain.services.IMenuService;
+import com.kritsit.casetracker.shared.domain.model.Staff;
+
+import javafx.collections.FXCollections;
+import javafx.fxml.FXML;
+import javafx.scene.control.Alert;
+import javafx.scene.control.Button;
+import javafx.scene.control.PasswordField;
+import javafx.scene.control.Alert.AlertType;
+
+public class ChangePasswordController {
+    
+    private Staff user;
+    private IMenuService menuService;
+    
+    public ChangePasswordController(Staff user, IMenuService menuService){
+        this.user=user;
+        this.menuService=menuService;
+    }
+    
+    public void initialize(){
+        changePasswordButton.setOnAction(event->{
+            changePassword(this.user, this.menuService);
+        });
+    }
+
+    private void changePassword(Staff user, IMenuService menuService){
+        Alert alert;
+        if(currentPasswordField.getText().equals("")||newPasswordField.getText().equals("")){
```

Comment:
Rather here call `curentPasswordField.getText().trim().isEmpty()` and the same for newPasswordField